ajax/dropdownReference.php

```diff
@@ -65,8 +65,8 @@
                                  '../ajax/dropdownReference.php',
                                  $params);
 
-// Get price
-} elseif (isset($_POST['reference_id'])) {
+   // Get price
```

Comment:
Well, our CS implies indentation, so yes, this is required (changes are from phpcbf). I guess this comment is misplaced, it should be inside the else (with its current indentation).